realm/src/main/java/io/realm/internal/TableQuery.java

```diff
@@ -402,70 +402,70 @@ public TableQuery notEqualTo(long columnIndex[], String value){
     }
     protected native void nativeNotEqual(long nativeQueryPtr, long columnIndex[], String value, boolean caseSensitive);
 
-    public TableQuery beginsWith(long columnIndex, String value, boolean caseSensitive){
+    public TableQuery beginsWith(long columnIndices[], String value, boolean caseSensitive) {
```

Comment:
If it's ever going to be public - and it likely is, then the consistency should be that there is a shortcut method for a single sort as that is mostly used. And that goes for all sort methods. The consistency would then be between the high object interface and the dynamic interface. But I'm ok with leaving it for now.